css/style.css

```
@@ -97,45 +101,45 @@ html {
   padding-bottom: 2rem;
 }
 
-.promo-slogan {      
+.promo-slogan {
   width: inherit;
   height: 25rem;
-  margin-top: 10rem;          
+  margin-top: 10rem;
 }
 
-.slogan {    
+.slogan {
   width: inherit;
-  padding: 0 2rem;  
-  font-family: "Yeseva One", sans-serif;       
+  padding: 0 2rem;
+  font-family: "Yeseva One", sans-serif;
   text-align: center;
   font-size: 4.8rem;
-  font-weight: 100;        
-  color: white;   
-  text-shadow: 1px 1px rgba(0, 0, 0, 0.5); 
+  font-weight: 100;
+  color: white;
```

Prefer HEX colors over text ones
```suggestion
  color: #fff;
```

---

fixed